src/connectors/sink.rs

```diff
@@ -751,16 +762,19 @@ where
                             self.merged_operator_meta.merge(event.op_meta.clone());
                             let transactional = event.transactional;
                             let start = nanotime();
-                            let res = self
-                                .sink
-                                .on_event(
-                                    port.borrow(),
-                                    event,
-                                    &self.ctx,
-                                    &mut self.serializer,
-                                    start,
-                                )
-                                .await;
+                            let res = if port == "control" {
```

Comment:
would be nice to `const` this out like `IN`, `OUT`, `ERR` so if we ever change it we can do it easily :) also saves me from typos